hugr-py/tests/test_hugr_build.py

```diff
@@ -404,3 +405,19 @@ def test_option() -> None:
     dfg.set_outputs(b)
 
     validate(dfg.hugr)
+
+
+def test_toposort() -> None:
+    f = Function("prepare_qubit", [tys.Bool, tys.Qubit])
+    [b, q] = f.inputs()
+
+    q = f.add_op(H, q).out(0)
+    b0 = f.add_op(Not, b)
+    b1 = b
+
+    f.set_outputs(q, b0, b1)
+
+    validate(Package([f.hugr], [QUANTUM_EXT]))
+
+    sorted_nodes = f.hugr.get_sorted_nodes(f.hugr.module_root)
+    assert len(list(sorted_nodes)) == len(list(f.hugr))
```

Comment:
This is a bad test. Will replace with asserting that the sort is a valid topological ordering